youtube_dl/extractor/mixcloud.py

```diff
@@ -122,3 +120,31 @@ def _real_extract(self, url):
             'view_count': view_count,
             'like_count': like_count,
         }
+
+class ServerAttemptSequence:
```

Comment:
It's ok, but it still doesn't work with the test urls. Which urls are you downloading?